fabric-screen-handler-api-v1/src/main/java/net/fabricmc/fabric/api/screenhandler/v1/package-info.java

```diff
@@ -38,6 +38,18 @@
  * Screen handlers can be opened using
  * {@link net.minecraft.entity.player.PlayerEntity#openHandledScreen(net.minecraft.screen.NamedScreenHandlerFactory)}.
  * Note that calling it on the logical client does nothing. To open an extended screen handler, the factory passed in
- * should be an {@link net.fabricmc.fabric.api.screenhandler.v1.ExtendedScreenHandlerFactory}.
+ * should be an {@link net.fabricmc.fabric.api.screenhandler.v1.ExtendedScreenHandlerFactory}, or a
+ * {@link net.minecraft.screen.SimpleNamedScreenHandlerFactory} that wraps such factory.
+ *
+ * <h2>Overwriting screen handlers</h2>
+ * You might have noticed that calling {@code openHandledScreen} while on another screen will move
```

Comment:
```suggestion
 * You might have noticed that calling {@link net.minecraft.entity.player.PlayerEntity#openHandledScreen(net.minecraft.screen.NamedScreenHandlerFactory) openHandledScreen} while on another screen will move
```
This makes it a link and iirc mercury remaps the reference too so it can be navigated to in mojmap envs